test/e2e/framework/framework.go

```diff
@@ -148,51 +136,31 @@ func IngressNginxDescribe(text string, body func()) bool {
 	return Describe("[nginx-ingress] "+text, body)
 }
 
-// GetNginxIP returns the IP address of the minikube cluster
-// where the NGINX ingress controller is running
+// GetNginxIP returns the number of TCP port where NGINX is running
```

Comment:
It's still returning an IP address, isn't it?

Reply:
Yes, the ingress-nginx service cluster IP